gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -5,12 +5,16 @@ import org.gradle.api.Project
 
 public class AndroidCommandPluginExtension {
 
+    static final String CATEGORIES_DEFAULT = ''
```

Comment:
What a default is an empty category? Would that work at all with the '-c' ? 
Should be inlined imho.